Source/JavaScriptCore/jsc.cpp

```diff
@@ -194,7 +194,11 @@ NO_RETURN_WITH_VALUE static void jscExit(int status)
         }
     }
 #endif // ENABLE(DFG_JIT)
+#if OS(WINDOWS)
+    TerminateProcess(GetCurrentProcess(), status);
```

Comment:
You might need to `#include <processthreadsapi.h>`?

Reply:
I guess `windows.h` is enough, but let's see the build on EWS. And if it fails, then let's add it.

Reply:
WinCairo build bot get built. So yeah, `windows.h` was enough. Landing.